client/java/src/main/java/io/openlineage/client/transports/composite/CompositeConfig.java

```diff
@@ -3,16 +3,18 @@
 /* SPDX-License-Identifier: Apache-2.0
 */
 
-package io.openlineage.client.transports;
+package io.openlineage.client.transports.composite;
```

Comment:
@mobuchowski @JDarDagran  This PR moves transports into separate packages which, in my opinion, clears the burden within the codebase. On the other hand, this may be seen as breaking public API and potential java client users other than spark and flink integration will have to modify their `import` in case they manually instantiate transport classes. Let me know if you have other opinion on that.

Spark and Flink users, using Spark/Flink conf or yaml files, won't see the difference.